### ISSUE TITLE
Getting syntax error when trying to run 'gulp init'

### ISSUE THREAD

**Issue by joshmfranklin** [uswds/uswds-gulp#8]
After setting up my gulpfile.js I am getting a syntax error for `gulp-notify`

```
[12:11:35] gulp-notify: [Error in notifier] SyntaxError in plugin "gulp-notify"
Message:
    Unexpected token { in JSON at position 129
```

I am unable to continue the gulp process due to this error.

**Comment by thisisdano** [comment]
Could be related to https://github.com/mikaelbr/gulp-notify/issues/129 — there may be nothing we can do until get more info from this upstream issue. I can perhaps uninstall `gulp-notify` in the interim.

**Comment by thisisdano** [comment]
As a temporary fix, we can just remove the `notify` from the pipe:

replace this in `gulpfile.js`, (starting at line 107):
```
    .pipe(gulp.dest(`${CSS_DEST}`))
    .pipe(notify({
      "sound": "Pop" // case sensitive
    }));
```

with this:
```
    .pipe(gulp.dest(`${CSS_DEST}`));
```

**Comment by kdimatteo** [comment]
This should get fixed in the upstream, but it doesn’t look like there’s much activity there. Seems easiest to drop from here since its breaking and only used to play “pop” sound on build complete.

Fix is in v0.0.2

**Comment by thisisdano** [comment]
`gulp-notify` removed in https://github.com/uswds/uswds-gulp/pull/9 (thanks @kdimatteo) so I'm going to consider this issue fixed. I'll make a new issue around reinstating `gulp-notify` that will be contingent on the upstream fix.
